rsync.te

```diff
@@ -53,7 +53,7 @@ gen_tunable(rsync_export_all_ro, false)
 ##	be labeled public_content_rw_t.
 ##	</p>
 ## </desc>
-gen_tunable(allow_rsync_anon_write, false)
+gen_tunable(rsync_anon_write, false)
```

Comment:
Please do not rename. While the new name is better, we don't have a way to preserve compatibility.